bundles/communities-ugc-migration/src/main/java/com/adobe/communities/ugc/migration/importer/ScoresSimpleImportServlet.java

```diff
@@ -0,0 +1,123 @@
+package com.adobe.communities.ugc.migration.importer;
+
+import com.adobe.granite.security.user.UserPropertiesManager;
+import com.adobe.granite.security.user.UserPropertiesService;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonToken;
+import org.apache.felix.scr.annotations.*;
+import org.apache.felix.scr.annotations.sling.SlingServlet;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.request.RequestParameter;
+import org.apache.sling.api.resource.*;
+import org.apache.sling.api.servlets.SlingAllMethodsServlet;
+import com.adobe.cq.social.scoring.api.ScoringService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.jcr.RepositoryException;
+import javax.servlet.ServletException;
+import java.io.IOException;
+import java.io.InputStream;
+
+
+@SuppressWarnings("serial")
+@SlingServlet(paths = "/services/social/scores/simple-import",
+            methods = "POST",
+            metatype = true)
+@Properties({
+    @Property(name = "service.vendor", value = "UGC Migration Profile Scores Migrate"),
+    @Property(name = "service.description", value = "Accepts a json file containing profile scores and applies them to stored profiles")
+})
+public class ScoresSimpleImportServlet extends SlingAllMethodsServlet {
+        
+    private static final Logger log = LoggerFactory.getLogger(ScoresSimpleImportServlet.class);
+    
+    @Reference
+    private ResourceResolverFactory resourceResolver;
+
+
+    @Reference(cardinality = ReferenceCardinality.MANDATORY_UNARY, policy = ReferencePolicy.STATIC)
+    private UserPropertiesService userPropertiesService;
+
+    @Reference
+    private ScoringService scoringService;
+
+
+    /**
+    *The post operation accepts a json file, parses it and applies the profile scores to local profiles
+    * @param request - the request
+    * @param response - the response
+    * @throws javax.servlet.ServletException
+    * @throws java.io.IOException
+    */
+    @Override
+    protected void doPost(SlingHttpServletRequest request, SlingHttpServletResponse response)
+            throws ServletException, IOException {
+
+        ResourceResolver resolver = request.getResourceResolver();
+        UGCImportHelper.checkUserPrivileges(resolver, resourceResolver);
+        final RequestParameter[] fileRequestParameters = request.getRequestParameters("file");
+
+        String resourcePath = "";
+        if(!request.getParameter("path").equals(null) && !request.getParameter("path").equals("")){
```

Comment:
Does this throw a NPE when the path is null? I think you want to do:
`if(request.getParameter("path") != null && !request.getParameter("path").equals(""))`